data.tf

```diff
@@ -1,3 +1,8 @@
+provider "mcaf" {
```

Comment:
Lets add this to main.tf then its consistent with the place where audit/logging provider is stored. main == management